docs/varmap.md

```diff
@@ -0,0 +1,27 @@
+Variable name mapping.
```

Comment:
I have an idea, why not include the short variable names in the documentation to each variable in `params.rs` and `setup.rs` instead of this file? The documentation can easily get out of date and it's harder to find, whereas it is easy to open `params.rs` and read the mappings there.

Reply:
Good idea!
I had a similar concern. For example, `n_p` exists in both `setup` and `params`. But, I could only link to one of them in this structure. Also, as I said in my comment, private variables and functions are not documented by the default setting. 
However, having a separate page for mapping is also useful while reading the documentation. Especially, when we have more complex doc pages, the reader might need to jump to the mapping. In addition, for the private ones, a reader cannot see any definition related to them on `docs.rs` when we publish the crate, without cloning the code. 

So, my suggestion is to keep this mapping and also have a description for each variable on its own page.

WDYT? @tolikzinovyev @rrtoledo  @djetchev

Reply:
> when we have more complex doc pages, the reader might need to jump to the mapping

could you give an example?

> for the private ones, a reader cannot see any definition related to them on docs.rs when we publish the crate

I mostly had in mind someone who is looking at the code and trying to understand the calculations. For non-public variables, they could also have a comment with the short name.